.github/workflows/ci.yml

```diff
@@ -0,0 +1,161 @@
+name: CI
```

Comment:
As I mentioned in Issue #1060, you're going to want to factorize the CI into separate workflows. GHA was not designed to have everything be shoved into a single config like Travis or Circle.